src/lib/contents/dropdown.ts

```diff
@@ -66,6 +66,23 @@ export const docsLinks: {
     description: "How to use Gitpod with any project.",
   },
 ];
+export const platformLinks: {
+  text: string;
+  href: string;
+  description: string;
+}[] = [
+  {
+    text: "Cloud",
+    href: "#",
```

Comment:
```suggestion
    href: "/cloud",
```

Accept this suggestion & merge PR after the merge of #3814